### ISSUE TITLE
`isolabel` should be sanitized

### ISSUE THREAD

**Issue by b0ch3nski** [ValdikSS/windows2usb#34]
`isolabel` can contain chars that `mkfs.vfat` cannot accept

https://github.com/ValdikSS/windows2usb/blob/9f855faabd9f33998b3c773927311510877f15a6/windows2usb#L252

```
mkfs.vfat: Labels with characters *?.,;:/\|+=<>[]" are not allowed
```

**Comment by ValdikSS** [comment]
I assume you have a custom ISO file. Custom ISO files are generally not supported as they may contain many modifications as Windows installer system is pretty flexible, which are not handled by windows2usb.

**Comment by b0ch3nski** [comment]
Agreed, but replacing the not allowed chars would be pretty much simple and protect from at least this kind of issue ;)

**Comment by b0ch3nski** [comment]
> @b0ch3nski What did you end up using to create the USB?

@Manouchehri Monkey-patch the code - remove these lines:

```
    isolabel="$(get_iso_name "$isopath")"
    if [ $? -ne 0 ]
    then
        isolabel=""
    fi
```

It will create the empty ISO label which will avoid the FAT32 limitations.

**Comment by ValdikSS** [comment]
Fixed in 0.2.2
